include/mata/nfa.hh

```diff
@@ -228,6 +270,7 @@ private:
     ///
     /// These states are used in the transition relation, either on the left side or on the right side.
     /// The value is always consistent with the actual number of states in the transition relation.
+    // TODO: the name of this is very bad, lets find another. What is it actually? What about tgt_domain_size, target_domain_size, domain_size.
```

Comment:
Maybe `state_domain_size`?

Reply:
could be, but better getting rid of it completely